### PR TITLE
Updated the UG.

### PR DESCRIPTION
**Pull request by ElysiaTanZY**

### CODE REVIEW

**Comment by ElysiaTanZY** [comment]
Please look through and help me spot any mistakes in the commands or anything that I missed out O: Thanksss!!

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -43,9 +44,12 @@ image::Ui.png[width="790"]
 e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 .  Some example commands you can try:
 
-* *`(ab)list`* : lists all contacts
-* **`(ab)add`**` n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : adds a contact named `John Doe` to the Address Book.
```
Comment (Vanessa1098) [inline docs/UserGuide.adoc:47]: You might have left out the (ab)list command in the UG.

Comment (ElysiaTanZY) [reply docs/UserGuide.adoc:47]: So (ab)list just lists all contacts then (ab)find allows users to list just specific contacts? Will it be considered overlapping commands though? O:

Comment (ElysiaTanZY) [reply docs/UserGuide.adoc:47]: Okay I added it back in already!

**Pull request merged**